lib/config.js

```diff
@@ -20,14 +22,51 @@ function Config () {
 	for (var key in config) {
 		this[key] = config[key];
 	}
+	
+	if (mode === 'worker'){
```

Comment:
Small comment, but worth mentioning for when I hook up JSHint.. Styling I've adopted for statements with brackets is adding a space like this `if (mode === worker) {`